cmd/svcat/output/version.go

```diff
@@ -24,9 +24,9 @@ import (
 // WriteVersion prints a list of bindings.
 func WriteVersion(w io.Writer, client, server string) {
 	if client != "" {
-		fmt.Fprintf(w, "client: %s\n", client)
+		fmt.Fprintf(w, "Client Version: %s\n", client)
```

Comment:
Just like kubectl version command:
https://github.com/kubernetes/kubernetes/blob/60cd056cb443c23b53240d274df4d5abc85c7aaf/pkg/kubectl/cmd/version.go#L121-L124

Reply:
good idea! Yes that would work well.